### PR TITLE
PLP Specification Editor

### PR DESCRIPTION
**Pull request by anthonysena**
Atlas changes related to issue OHDSI/WebAPI#224. Must be tested with the WebAPI plp branch submitted through PR: OHDSI/WebAPI#248.

### CODE REVIEW

**Comment by anthonysena** [comment]
Sure! I've pushed some commits to clean up the formatting so let me know if you face any problems.

File: js/makeTable.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
+function makeTable() {
```
Comment (chrisknoll) [inline js/makeTable.js]: This should be wrapped in an AMD module if it's going to be some reusable piece of code.

File: js/makeTable.js [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+	
+		if(typeof sort_by !== 'undefined'){
+			// if we have a sort_by column, format it according to datatables.
+			sort_by[0] = colnames.indexOf(sort_by[0]); //colname to col idx
+			sort_by = [sort_by]; //wrap it in an array
+		}
+		
+
+	  // Apply DataTable formatting: https://www.datatables.net/
+	  $(document).ready(function() {
```
Comment (chrisknoll) [inline js/makeTable.js]: I'm curious where we want to use a DOMReady function, shouldn't this be handled by databinding or some other event trigger?

**Comment by anthonysena** [comment]
I've removed makeTable.js as a dependency as it is already embedded into [plp-inspector](https://github.com/OHDSI/Atlas/blob/69c049f4ba5b13d5e16bde278d977357c86dea02/js/components/plp-inspector.html)

**Pull request merged**